ethtx_ce/frontend/templates/index.html

```diff
@@ -161,30 +175,46 @@ <h2>About EthTx</h2>
             <p>Everybody is invited to contribute to the codebase (<a href="https://github.com/EthTx/ethtx"
                                                                       class="link">GitHub</a>) via Pull Request or
                 Issues and anyone can launch their own instances of EthTx.</p>
+            <p id="ethtx_version" class="version">Version: {{ ethtx_version }}</p>
         </div>
         <div class="panel-container">
             <h2>About this implementation</h2>
             <p>Source code of this Community Edition of EthTx is also free and open source
                 (<a href="https://github.com/EthTx/ethtx_ce" class="link">GitHub</a>).</p>
             <p>Use of this implementation is under the <a href="terms" class="link">Terms of Service</a> and <a
                     href="privacy" class="link">Privacy Policy</a></p>
+            <p id="ethtx_ce_version" class="version">Version: <a class="link" href={{ repo_version }}> ethtx_ce</a></p>
         </div>
     </div>
+    {% include './partials/footer.html' %}
 </div>
-{% include './partials/footer.html' %}
-<script>
+
+<script type="text/javascript">
     document.addEventListener("DOMContentLoaded", function () {
         function trackDecode() {
             // fathom.trackGoal('PR1BHWAQ')
         }
 
         document.getElementById('tx_submit').addEventListener('click', trackDecode)
     });
-</script>
-<script type="text/javascript">
+
     function handleValueChange(value) {
         document.getElementById('tx_submit').disabled = value.trim() === "";
     }
+
+    function hideNullVersions() {
+        const version = "Version: "
+        const ethtx_content = document.getElementById('ethtx_version').textContent.replace(version, "").trim();
+        const ethtx_ce_content = document.getElementById('ethtx_ce_version').textContent.replace(version, "").trim();
+
+        if (!ethtx_content) {
+            document.getElementById('ethtx_version').style.visibility = "hidden";
+        }
+
+        if (!ethtx_ce_content) {
+            document.getElementById('ethtx_ce_version').style.visibility = "hidden";
+        }
+    }
```

Comment:
Instead of using js for this you can use jijnja2 if else statement https://stackoverflow.com/questions/22655031/jinja2-how-to-put-a-block-in-an-if-statement
This way output page would look better for people who have js disabled and crawler bots like google.